app/models/npq_application.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class NPQApplication < ApplicationRecord
+  attr_accessor :current_user
```

Comment:
I would name this field differently, maybe `admin_user`? Or different approach - field that store `bool` value that application can be modified. Approving anyway.

Reply:
Actually, `current_user` instance does not exist within the scope of this modal. That's why I used this approach to pass it in this way.